.github/actions/bicep-to-arm-template-diff/action.yaml

```diff
@@ -0,0 +1,36 @@
+name: Bicep to ARM template diff
+description: Bicep build to ARM template diff 
+inputs:
+  source-file-path:
+    description: Bicep script path
+    required: true
+  target-file-path:
+    description: Arm template file path
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Build bicep into arm
+      shell: bash
+      run: |
+        az bicep build --file ${{ inputs.source-file-path }} --stdout | jq -S . > source.json
+        cat source.json
```

Comment:
this is going to generate a lot of logs, I think it's fine to not cat the entire ARM since we have the diff below, and if something is wrong we could just reproduce locally in the branch?

Reply:
oh my bad, I did that when I was debugging. Let me quickly fix that